### PR TITLE
refactor: buy ticket to register

### PR DESCRIPTION
**Pull request by basiima**

### CODE REVIEW

File: components/BuyTicketSection.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function BuyTicketSection() {
                     <p className="text-base p-0 font-medium">Tickets can be bought by following the link below.</p>
                     <div className="py-12">
                         <Link className="bg-[#ffe135] text-white font-bold py-3 px-10 m-2 border rounded-3xl" href={QUICKET_LINK}>
-                            Buy a Ticket
+                            Register
```
Comment (HassanBahati) [inline components/BuyTicketSection.js:13]: @basiima does this mean we are doing away with `buy ticket` for `register` ?

Comment (basiima) [reply components/BuyTicketSection.js:13]: Yes as agreed in the meeting, until the payments issues are resolved

Comment (HassanBahati) [reply components/BuyTicketSection.js:13]: But the link remains...doesnt that mean we need to entirely remove the link too?

**Comment by basiima** [comment]
I have updated the link. It directs one to the quicket page for free tickets

File: components/Navbar.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default function Navbar() {
             href={QUICKET_LINK}
           >
             {" "}
-            Register
+            Get Ticket
```
Comment (HassanBahati) [inline components/Navbar.js:46]: @basiima does this mean we standardizing as `get ticket` than `buy ticket` ?

I feel `get ticket` is better.
This should reflect also on the navbar button

Comment (basiima) [reply components/Navbar.js:46]: Yes

Comment (HassanBahati) [reply components/Navbar.js:46]: Awesome

**Review by HassanBahati — APPROVED** [review]

**Pull request merged**